.buildbot.sh

```diff
@@ -22,6 +22,13 @@ export PATH=${CARGO_HOME}/bin/:$PATH
 
 rustup toolchain install nightly --allow-downgrade --component rustfmt
 
+# Check that if we build the system without support for testing and debugging
```

Comment:
I'm struggling to parse this sentence.

Reply:
I struggled to write it too. I'll see if I can come up with something better.

Reply:
Thanks!

Reply:
Thanks!

Reply:
Ditto. It might help breaking this up into multiple sentences.

Reply:
Does d0293028143f4a95bdeb783cf67434486f9b9896 help?